docs/reference/search/field-caps.asciidoc

```diff
@@ -77,6 +77,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailab
   (Optional, Boolean) If `true`, unmapped fields that are mapped in one index but not in another are included in the response. Fields that don't have any mapping are never included.
   Defaults to `false`.
 
+`include_fields_with_no_value`::
```

Comment:
would `include_empty_fields` help making the name shorter, without removing bits of important info from it?

Reply:
Done!